app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,81 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class TweetsController < ApplicationController
+  get '/tweets' do
+    if !Helpers.is_logged_in?(session)
+      redirect to '/login'
+    else
+      @user = User.find_by_id(session[:user_id])
+      @tweets = Tweet.all
+      erb :'/tweets/index'
+    end
+  end
+
+  get '/tweets/new' do
+    if Helpers.is_logged_in?(session)
+      erb :'/tweets/new'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if params[:content].empty?
+      redirect to '/tweets/new'
+    else
+      @tweet = Tweet.create(content: params[:content], user_id: session[:user_id])
+      # if params[:phone_number] != ""
```

Comment:
This goes along with a comment 2 down: associating @tweet with the current_user, as you've suggested, would be a more robust (? better word?) way to prevent edits from other users?